docs/tutorials/byproduct_recovery_determination.ipynb

```diff
@@ -0,0 +1,706 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "0a29c413-0a01-4f16-bf26-29a60dae142a",
+   "metadata": {},
+   "source": [
+    "# __Byproduct Recovery Framework Tutorial__\n",
+    "\n",
+    "The purpose of this tutorial is to introduce the __byproduct recovery decision framework__ (``determine_byproduct_recovery.py``), which is based on the criterion of whether the total annual revenue is non negative. This framework serves as a supporting tool to help determine whether certain critical minerals (CMs) and/or rare Earth elements (REEs) should be recovered. \n",
+    "\n",
+    "While this tutorial focuses on revenue-based decisions, byproduct recovery decisions can also be guided by other criteria—such as the abundance of the byproduct or its current market value.\n",
+    "\n",
+    "The decision tree used in this tutorial is referenced from Chapter 8 of the book _Chemical engineering design: principles, practice and economics of plant and process design_ by Towler, Gavin, and Ray Sinnott, Butterworth-Heinemann, 2021.\n",
+    "![My Image](byproduct_recovery_determination_tree.png)\n",
+    "\n",
+    "The framework includes a built in hypothesis example that results in positive revenue. In this tutorial, we will also demonstrate how to apply this framework to assess whether __lithium__ and __colbolt__ should be recovered through a dialfiltration process. \n",
+    "\n",
+    "To use the byproduct recovery framework effectively, a byproduct recovery process must already be constructed. This is necessary to determine the __annualized cost__ and __annual revenue__ associated with recovery. Once the process model is built, users can import it—along with its corresponding cost and revenue—and apply the byproduct recovery framework to decide whether or not the byproduct should be recovered.\n",
+    "\n",
+    "__Learning Objectives__: By the end of this tutorial, users will be able to:\n",
+    "- Decide and import the necessary Python packages\n",
+    "- Build a process flowsheet using a case study: __lithium-colbalt recovery via diafiltration__\n",
+    "- Apply the __byproduct recovery decision framework__"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d56a0147-4343-4ebf-8dc1-b7eb004b455b",
+   "metadata": {},
+   "source": [
+    "## __Step 1: Import the Necessary Packages__\n",
+    "\n",
+    "First, import the necessary Pyomo, IDAES, and PrOMMiS packages. These will be implemented at various stages throughout the tutorial.\n",
+    "\n",
+    "If you haven't installed these packages yet, you can do so using the following commands:"
```

Comment:
Generally, it's best to avoid using second person "you" or "your" in tutorials, so rephrase these kinds of sentences to "if these packages are not installed, this may be done using the following commands".